### DISCUSSION TITLE
Solocraft Mod - Docker

### DISCUSSION THREAD

**Discussion by CZAlmon** [azerothcore/azerothcore-wotlk#11988] [Modules]
This is a stupid question, but on both the azeorthcore docker webpage and on the solocraft github README.md they both state to "simply place the module directory inside of the /azerothcore-wotlk/modules directory". Then rebuild. My question is put exactly what in that directory? I cloned the solocraft mod from github and then put that folder in the azerothcore-wotlk/modules directory and rebuilt it, but it did not work. The mod is not active in my game. I did put the conf file in the correct directory, as per azerothcore's installation directions. I am just wondering if I should have put the raw src/data files from solocraft into the mod directory.

**Comment by acidmanifesto** [comment]
Did u rerun cmake before rebuilding?

**Reply by CZAlmon** [reply]
I didn't run cmake, how do you run that? I did rerun the docker build as per the instructions here: https://www.azerothcore.org/wiki/install-with-docker#adding-modules
